### DISCUSSION TITLE
blog/release-of-tailwind-nextjs-starter-blog-v2

### DISCUSSION THREAD

**Discussion by giscus** [muhghazaliakbar/ngoding#30] [Announcements]
# blog/release-of-tailwind-nextjs-starter-blog-v2

Release of Tailwind Nextjs Starter Blog template v2.0, refactored with Nextjs App directory and React Server Components setup.Discover the new features and how to migrate from V1.

https://www.ngoding.net/blog/release-of-tailwind-nextjs-starter-blog-v2.0

**Comment by muhghazaliakbar** [comment]
Adakah?
